discovery-provider/plugins/notifications/src/sns.ts

```diff
@@ -138,3 +187,12 @@ export const sendPushNotification = async (
     })
   }
 }
+
+export const sendPushNotificationBatch = async (
```

Comment:
actually does this get used anywhere too??

Reply:
i didn't notice this getting used anywhere but i'll take another look!